### ISSUE TITLE
Got error: "isPlainObject is not a function" in version >3.0.0

### ISSUE THREAD

**Issue by quertenmont** [mariocasciaro/object-path-immutable#47]
Hello, 

In recent version of the package 
I get the error

```
TypeError: isPlainObject is not a function
    at deepMerge (index.js:103)
```
When I retrograde the version to 3.0.0, everything works fine.
Would be nice to correct this.

All the best,
Loic

**Comment by mariocasciaro** [comment]
Hi folks, what environment are you using? Node or Browser? are you using any bundler/compiler?

**Comment by vladimir-shalaev22** [comment]
> Hi folks, what environment are you using? Node or Browser? are you using any bundler/compiler?

Thanks for response! I'm using Create React App. Bug occurs when I test app in browser in dev mode. But while I test code that uses object-path-immutable with Jest (Node) - all works nice

**Comment by mariocasciaro** [comment]
Should be fixed in 4.0.0
